src/coverlet.core/Instrumentation/Instrumenter.cs

```diff
@@ -67,20 +70,119 @@ private void InstrumentModule()
                 using (var module = ModuleDefinition.ReadModule(stream, parameters))
                 {
                     var types = module.GetTypes();
+                    AddCustomModuleTrackerToModule(module);
+
                     foreach (TypeDefinition type in types)
                     {
                         var actualType = type.DeclaringType ?? type;
                         if (!actualType.CustomAttributes.Any(IsExcludeAttribute)
+                            && actualType.Namespace != "Coverlet.Core.Instrumentation.Tracker"
```

Comment:
Why not add an `ExcludeFromCodeCoverage` attribute to all the methods in that class?

Reply:
I see that you already removed it 👍 Thanks, it was not needed as you noticed